node_modules/oae-authz/tests/test-groups.js

```diff
@@ -15,10 +15,12 @@
 
 var _ = require('underscore');
 var assert = require('assert');
+var shortid = require('shortid');
```

Comment:
Why does this one get camel cased while all the other external deps are lowercased? I'd suggest just renaming the others to `shortid` so the style is more predictable